public/projects.html

```diff
@@ -0,0 +1,54 @@
+<head>
+  <title>Lila Garcia - Projects
+  </title>
+</head>
+
+<header>
+<h1>Projects</h1>
+<nav>
+  <a href="/">Home</a>&nbsp;&nbsp;&nbsp;
+  <a href="about">About</a>&nbsp;&nbsp;&nbsp;
+  <a href="blog">Blog</a>&nbsp;&nbsp;&nbsp;
+</nav>
+<a name="top"></a>
+</header>
+
+<body>
+  <h2>Ada Developers Academy</h2>
+  <h5>Track my progression as I go from newb, to sorta-okay progammer:</h5>
+  <section>
+    <h4>Week 4</h4>
```

Comment:
Your semantic markup here is spot on. This whole page looks great. Fantastic job.